pkg/apis/applicationset/register.go

```diff
@@ -0,0 +1,13 @@
+package applicationset
+
+const (
+	// API Group
+	Group string = "appset.argoproj.io"
```

Comment:
This  would change the ApiVersion for ApplicationSets from `argoproj.io` to `appset.argoproj.io`. If we do not update this, the ClientSet interface is not generated properly.

To avoid changing the ApiVersion, the alternative might be to move all files from `pkg/apis/applicationset/v1alpha1` to `pkg/apis/application/v1alpha1`. This would have changes to a lot of imports throughout the repository and possibly other repositories referencing argo-cd too.

Let me know if anyone has other opinions around this.

Reply:
(Repeating from above) I think we have to go with the file-moving option. :-(